lib/pry/method/weird_method_locator.rb

```diff
@@ -26,13 +26,17 @@ class << self
         # @param [Binding] b
         # @return [Boolean]
         def normal_method?(method, b)
-          method && (method.source_file && method.source_range rescue false) &&
-          File.expand_path(method.source_file) == File.expand_path(b.eval('__FILE__')) &&
-          method.source_range.include?(b.eval('__LINE__'))
+          if method and method.source_file and method.source_range
+            binding_FILE, binding_LINE = b.eval('__FILE__'), b.eval('__LINE__')
```

Comment:
I think `binding_file` is nicer than `binding_FILE` though i can see what you were going for here, i still personally prefer `binding_file`